src/operation/drop_collection/test.rs

```diff
@@ -15,6 +15,7 @@ fn build() {
             w: Some(Acknowledgment::Custom("abc".to_string())),
             ..Default::default()
         }),
+        ..Default::default()
```

Comment:
I think we've talked in the past about removing these tests when they break rather than continuing to maintain them -- should we do that here?

Reply:
Whoops, that had dropped out of my head - removed the test.